[W-16730299] Update amazon-secrets-manager-examples.adoc (#2772)
mulesoft/docs-connectors: amazon-secrets-manager-properties-provider/1.0/modules/ROOT/pages/amazon-secrets-manager-examples.adoc

	</s3:config>
----

> It is recommended to avoid using the '$' special character in the values stored in the secret manager. If the '$' character is necessary, it is advised to use the variable in literal mode, which treats it as a string. After that, you can convert the entire value into JSON and extract the required information.

Reviewer: ```suggestion
Avoid using the `$` special character in values stored in the secret manager. If you must use the `$` character, use the variable in literal mode, which treats it as a string. After that, you can convert the entire value to JSON and extract the required information.
```
